WcaOnRails/config/locales/en.yml

```diff
@@ -507,6 +509,8 @@ en:
         on_the_spot_entry_fee_lowest_denomination: "If this is set to 0, a sentence will be displayed saying that on the spot registrations will be accepted for free."
         refund_policy_percent: "For now this number is informational only, refunds will have to be issued manually. If this is set to 0, a sentence will be displayed saying that registration fees won't be refunded under any circumstance."
         refund_policy_limit_date: "Date after which no more refunds will be issued."
+        waiting_list_deadline_date: "The date when waitlisted registrants will no longer be accepted. If you have a deadline for refunds, this time should not be before this."
```

Comment:
In the last sentence, WCAT realised that using "this" twice might be a bit confusing. So e.g. "[...] the deadline for being accepted should not be before the deadline for refunds" would be more clear though a bit longer.